src/components/Home/LatestActivity/BatchVotingWebView.tsx

```diff
@@ -21,12 +27,28 @@ const BatchVotingWebView = () => {
 				<Button
```

Comment:
it shoould be a link

Reply:
there is a button and on click of button it should either open loginModal or will redirect